event/forms.py

```diff
@@ -10,11 +10,33 @@ class Meta:
             'end_time': forms.DateTimeInput(attrs={'type': 'datetime-local'}),
         }
 
-class CreateGroupEventForm(forms.ModelForm):            
+class CreateGroupEventForm(CreateEventForm):            
```

Comment:
oh! is it something like refactor? what does this do ?

Reply:
It just reuses the prefious form since most of the fields are the same, that way we only need to do the widgets once